CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.33 [2021-05-18]
+
+_What's new?_
+
+- Added: FedRAMP Low Revision 4 benchmark (`steampipe check benchmark.fedramp_low_rev_4`)
+- Added: FedRAMP Moderate Revision 4 benchmark (`steampipe check benchmark.fedramp_moderate_rev_4`)
+
```

Comment:
I think there's another commit for the Lambda function bug fix, can you please include those as well?